### DISCUSSION TITLE
Is it possible to get the location of all error nodes from the treesitter parser

### DISCUSSION THREAD

**Discussion by IndianBoy42** [nvim-treesitter/nvim-treesitter#1831] [Q&A] [answered]
In lua code. I want to be able to see them all and jump to them using telescope or hop.nvim

**Comment by theHamsta** [comment, marked as answer]
To see them map `luaTSError` to a highlight you want them to have.

If you want to have their locations, you can launch a query with `(ERROR) @error`. The Neovim documentation contains information in how to launch queries and process the results.
